pkg/virtctl/scp/scp.go

```diff
@@ -80,9 +83,21 @@ func (o *SCP) Run(cmd *cobra.Command, args []string) error {
 	return o.nativeSCP(local, remote, toRemote)
 }
 
-func PrepareCommand(cmd *cobra.Command, clientConfig clientcmd.ClientConfig, opts *ssh.SSHOptions, args []string) (local templates.LocalSCPArgument, remote templates.RemoteSCPArgument, toRemote bool, err error) {
+type LocalArgument struct {
+	Path string
+}
+
+type RemoteArgument struct {
+	Kind      string
+	Namespace string
+	Name      string
+	Username  string
+	Path      string
+}
+
+func PrepareCommand(cmd *cobra.Command, clientConfig clientcmd.ClientConfig, opts *ssh.SSHOptions, args []string) (local LocalArgument, remote RemoteArgument, toRemote bool, err error) {
 	opts.IdentityFilePathProvided = cmd.Flags().Changed(ssh.IdentityFilePathFlag)
-	local, remote, toRemote, err = templates.ParseSCPArguments(args[0], args[1])
+	local, remote, toRemote, err = ParseArguments(args[0], args[1])
```

Comment:
```suggestion
	local, remote, toRemote, err = ParseTarget(args[0], args[1])
```

Might call this `ParseTarget` as well, for consistency with ssh. See my comments below.